Doppler.Sap.Test/BillingServiceTest.cs

```diff
@@ -382,5 +382,146 @@ public async Task BillingService_ShouldBeNotAddOneTaskInQueue_WhenUpdateBillingR
             queuingServiceMock.Verify(x => x.AddToTaskQueue(It.IsAny<SapTask>()), Times.Never);
             slackServiceMock.Verify(x => x.SendNotification(It.IsAny<string>()), Times.Once);
         }
+
+        [Fact]
+        public async Task BillingService_ShouldBeAddOneTaskInQueue_WhenCreateCreditNotesHasOneValidElement()
+        {
+            var billingValidations = new List<IBillingValidation>
+            {
+                new BillingForArValidation(Mock.Of<ILogger<BillingForArValidation>>()),
+                new BillingForUsValidation(Mock.Of<ILogger<BillingForUsValidation>>())
+            };
+
+            var sapConfigMock = new Mock<IOptions<SapConfig>>();
```

Comment:
This variable is not used, you can remove it

Reply:
Done